exercises/practice/bowling/.meta/config.json

```diff
@@ -19,6 +20,6 @@
       ".meta/src/Bowling.example.elm"
     ]
   },
-  "source": "Inspired by Uncle Bob's Bowling Game Kata",
+  "source": "The Bowling Game Kata at but UncleBob",
```

Comment:
Hi Jie, this text looks wrong, was it done by configlet?

If so I think we should update configlet, if not then I think the original text is ok.

Cheers, Cedd

Reply:
It was configlet, yes.
You're right, it doesn't look correct, but that's the data over on [problem-specs](https://github.com/exercism/problem-specifications/blob/main/exercises/bowling/metadata.toml), it would need to be changed over there.

Reply:
Actually, it DOES look correct:
http://butunclebob.com/ArticleS.UncleBob.TheBowlingGameKata

"but UncleBob" is the name of the site.

Reply:
Well, it doesn't look right! Maybe at butunclebob.com would be better. I'll merge this and create an issue on problem-specs ....